From 776477b3342e72b350c9f2c994365d1175e1cc3e Mon Sep 17 00:00:00 2001
From: denglei <denglei25@huawei.com>
Date: Mon, 26 Apr 2021 15:03:25 +0800
Subject: [PATCH] openeuler_defconfig: Enable hifc driver as module
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 776477b3342e72b350c9f2c994365d1175e1cc3e


driver inclusion
category: feature
bugzilla: 21

-----------------------------------------------------------------------

Hifc driver is used to support huawei Fibre Channel intelligent interface cards,
mainly including SP520, SP521, SP522, SP523, SP525, SP526 and SP527.

Enable hifc driver to be built as a module.

Signed-off-by: denglei <denglei25@huawei.com>
Reviewed-by: chenguangli <chenguangli2@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 ++
 arch/x86/configs/openeuler_defconfig   | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index e05176233584..07b4082459ac 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -2316,6 +2316,8 @@ CONFIG_QEDI=m
 CONFIG_QEDF=m
 CONFIG_SCSI_LPFC=m
 # CONFIG_SCSI_LPFC_DEBUG_FS is not set
+CONFIG_SCSI_HUAWEI_FC=m
+CONFIG_SCSI_FC_HIFC=m
 # CONFIG_SCSI_DC395x is not set
 # CONFIG_SCSI_AM53C974 is not set
 # CONFIG_SCSI_WD719X is not set
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 9f58fe9abd70..db039bbe8d29 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -2349,6 +2349,8 @@ CONFIG_QEDI=m
 CONFIG_QEDF=m
 CONFIG_SCSI_LPFC=m
 # CONFIG_SCSI_LPFC_DEBUG_FS is not set
+CONFIG_SCSI_HUAWEI_FC=m
+CONFIG_SCSI_FC_HIFC=m
 # CONFIG_SCSI_DC395x is not set
 # CONFIG_SCSI_AM53C974 is not set
 # CONFIG_SCSI_WD719X is not set
-- 
2.26.2

